conference.js

```diff
@@ -483,7 +487,19 @@ export default {
         let tryCreateLocalTracks;
 
         // FIXME the logic about trying to go audio only on error is duplicated
-        if (options.startScreenSharing) {
+        if (options.startAudioOnly) {
+            tryCreateLocalTracks
+                = createLocalTracks({ devices: ['audio'] }, true)
+                    .catch(err => {
+                        audioOnlyError = err;
+
+                        return [];
+                    });
+            // Enable audio only mode
+            if (!this.isAudioOnly()) {
```

Comment:
Yeah, since we are creating the audio only track here, there is no way that check returns false.